src/test/java/org/jacp/controller/QuestionControllerTest.java

```diff
@@ -0,0 +1,82 @@
+package org.jacp.controller;
+
+import org.jacp.dto.QuestionDto;
+import org.jacp.entity.QuestionEntity;
+import org.jacp.mapper.QuestionMapper;
+import org.jacp.service.QuestionService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+/**
+ * @author saffchen created on 10.08.2023
+ */
+@ExtendWith(MockitoExtension.class)
+class QuestionControllerTest {
+
+    static final String URL = "/api/v1/questions";
+
+    @Mock
+    private QuestionMapper questionMapper;
+
+    @Mock
+    private QuestionService questionService;
+
+    @InjectMocks
+    private QuestionController questionController;
+
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    public void setup(){
+        mockMvc = MockMvcBuilders.standaloneSetup(questionController).build();
+    }
+
+    @Test
+    void get() throws Exception {
```

Comment:
Minor: please give more detailed names to your methods.

Reply:
Fixed